src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv401.cql

```diff
@@ -628,10 +630,10 @@ define PregnancyConditions:
 //   C3F.Verified(C3F.ObservationLookBack([Observation: "Pregnancy status"], 42 weeks))
 
 // define MostRecentPregnancyObservation:
-//   C3F.MostRecent(PregnancyObservations)
+//    C3F.MostRecent(PregnancyObservations)
```

Comment:
```suggestion
//   C3F.MostRecent(PregnancyObservations)
```

Reply:
minor nitpick: I don't think this functionality has changed; leaving it as-is could make it easier to use git history to track changes